src/components/VisualizationStep.tsx

```diff
@@ -15,6 +15,7 @@ import { AlertVariant, Popover } from '@patternfly/react-core';
 import { CubesIcon, PlusIcon } from '@patternfly/react-icons';
 import { useAlert } from '@rhoas/app-services-ui-shared';
 import { Handle, Node, NodeProps, Position, useNodes } from 'reactflow';
+import {fetchStepDetails} from "@kaoto/api";
```

Comment:
tiny comment here but the spacing! to keep it consistent. also same comment about the double quotes vs single. I will create an issue to add prettier to part of the workflow so we don't have to worry about this.

Reply:
yeah, thanks! formatting the code from my IDE formats everything even your code so I'm doing this manually.